File: blog_vue/src/model/user/Author.ts
import UserInformation from "./UserInformation";

export default class Author extends UserInformation {
  articleCount: number;
  notableCount: number;
  fansCount: number;

  constructor() {
    super();
    this.articleCount = -1;
    this.notableCount = -1;
    this.fansCount = -1;
  }
}

File: blog_springBoot/src/main/java/com/synergism/blog/core/user/entity/Author.java
package com.synergism.blog.core.user.entity;

import lombok.Getter;
import lombok.Setter;

/**
 * 作者信息
 */
@Setter
@Getter
public class Author extends UserInformation {
    //所属文章数量
    private int articleCount;
    //关注数量
    private int notableCount;
    //粉丝数量
    private int fansCount;

    public Author() {
        super();
    }
}
